src/main/resources/view/MainWindow.fxml

```diff
@@ -24,42 +26,50 @@
       </stylesheets>
 
       <VBox>
-        <MenuBar fx:id="menuBar" VBox.vgrow="NEVER">
-          <Menu mnemonicParsing="false" text="File">
-            <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit" />
-          </Menu>
-          <Menu mnemonicParsing="false" text="Help">
-            <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
-          </Menu>
-        </MenuBar>
+            <HBox  >
```

Comment:
You could replace this `HBox` component with a StackPane which would allow you to center the logo using absolute positioning :)

        <StackPane>
          <children>
            <MenuBar fx:id="menuBar" prefHeight="35.0" minHeight="35.0" prefWidth="690.0" HBox.hgrow="ALWAYS">
                <Menu mnemonicParsing="false" text="File">
                  <items>
                    <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit" />
                  </items>
                </Menu>
                <Menu mnemonicParsing="false" text="Help">
                  <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
                </Menu>
            </MenuBar>
            <ImageView fitHeight="35.0" fitWidth="143.0" pickOnBounds="false">
              <image>
                <Image url="@../images/task_hub_logo.png" />
              </image>
            </ImageView>
          </children>
        </StackPane>